zilencer/lib/stripe.py

```diff
@@ -92,6 +92,14 @@ def get_upcoming_invoice(stripe_customer_id: int) -> stripe.Invoice:
         print(''.join(['"upcoming_invoice": ', str(stripe_invoice), ',']))  # nocoverage
     return stripe_invoice
 
+def extract_current_subscription(stripe_customer: stripe.Customer) -> stripe.Subscription:
```

Comment:
should be `Optional[stripe.Subscription]`?

Reply:
Since the type of Subscription is not defined yet mypy is throwing the following error.
```
zilencer/tests/test_stripe.py:245: error: Value of type "Optional[Any]" is not indexable
```
I have explained it in the commit message. Should be fixed when the stubs get merged.